jsettlers.graphics/src/jsettlers/graphics/map/MapContent.java

```diff
@@ -605,7 +605,7 @@ private static Action getActionForKeyboard(String keyCode) {
 			return new Action(EActionType.SPEED_TOGGLE_PAUSE);
 		} else if ("BACK".equalsIgnoreCase(keyCode)) {
 			return new Action(EActionType.BACK);
-		} else if ("+".equals(keyCode)) {
+		} else if ("]".equals(keyCode)) {
```

Comment:
Oops this should be on my "do not commit" part in idea :-)
This is another topic: at my mac the keycodes are different. The + at my keyboard is ] as keycode. I changed that (only for me) to be able to increase the speed.